HW05.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "HW05"
+output: github_document
+author: "Daniel Krochmal"
+date: "8/4/2020"
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+``` {r load libraries, echo = FALSE, message = FALSE}
+library(readr)
+library(knitr)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+```
+
+
+## Part 1
+
+
+``` {r import data, echo = FALSE, message = FALSE}
+Dad_mom <- read_tsv("dad_mom.txt") #Importing data using readr
+```
+
+``` {r tidy data}
+
+#I will use a sequence of gather(), separate() and spread() verbs to clean-up the datafame
+
+tidyDad_mom <- Dad_mom %>%
+  
+  gather(key = "var", value = "value",
```

Comment:
This was a clever way to tidy the data! I manipulated the columns _before_ calling `gather()`, but your method is easy to follow and very effective